### DISCUSSION TITLE
keybind text: how to send the <enter> key?

### DISCUSSION THREAD

**Discussion by zapman449** [ghostty-org/ghostty#4192] [Q&A] [answered]
Goal: have ghostty intercept cmd-s (super+s) and instead send `:w<enter>`.  This would make vim sessions "honor" the cmd-s for save file.

[redacted question, as I figured it out]

For future searchers, here's how I set it up (improvements welcome, particularly if I could do it with `csi` instead):

```
keybind = super+s=text:\x1b:w\n
```

**Comment by 00-kat** [comment, marked as answer]
> improvements welcome, particularly if I could do it with `csi` instead

*Super* late, but no; the way you discovered is the only method.
